build.proj

```diff
@@ -65,7 +65,23 @@
 
   <Import Project="dir.targets" />
   <Import Project="dir.traversal.targets" />
-  <Import Project="$(ToolsDir)clean.targets" />
+  <Import Project="$(ToolsDir)clean.targets" />  
+ 
+  <!-- 
+    This is to initialize the ProjectJsonFiles ItemGroup with the template files during the update dependencies target
+    only to prevent ValidateExactRestore from failing on just the regular project.json files.
+  -->
+  <Target Name="IncludeOnlyTemplateFilesForUpdate" BeforeTargets="UpdateDependencies">
+    <ItemGroup>
+      <ProjectJsonFiles Remove="*" />
```

Comment:
Is this line doing anything? It doesn't seem like it would match anything (I'd expect `Remove="@(ProjectJsonFiles)"`) and I can't find where any `ProjectJsonFiles` is defined in the current iteration.